contracts/honeylemon/MarketContractProxy.sol

```diff
@@ -30,6 +30,7 @@ contract MarketContractProxy is ReentrancyGuard, Ownable {
     address public HONEY_LEMON_ORACLE_ADDRESS;
     address public MINTER_BRIDGE_ADDRESS;
     address public COLLATERAL_TOKEN_ADDRESS; //imBTC
+    address public HONEY_MULTISIG;
```

Comment:
Do we need to explicitly maintain this address? Won't it always be equal to the owner of the `MarketContractProxy` contract?

Reply:
yup correct, I got ur comment now @chrismaree ... will remove that.